apps/website/src/pages/docs/inputgroup.mdx

```diff
@@ -9,17 +9,18 @@ group: inputs
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='input-inputgroup' />
-
 <LiveExample src='InputGroup.main.tsx'>
   <AllExamples.InputGroupMainExample client:load />
 </LiveExample>
 
+Input group class is a container to enhance an input by adding an icon, text, or a button in front or behind the input field as a "help text".
+
+Multiple add-ons are supported and can be mixed with [checkbox](checkbox), [radio](radio), [toggle switch](toggleswitch) input version. To learn more about when to use a toggle switch or a checkbox, please read [Checkbox vs Toggle Switch by Saadia Minhas](https://uxplanet.org/checkbox-vs-toggle-switch-7fc6e83f10b8).
```

Comment:
Remove link to "Checkbox vs Toggle Switch by Saadia Minhas" article. It's already referenced in both, checkbox and toggle switch.

Reply:
Hi, Ida, you should let the person who made the review resolve the issue. In this case, while you removed the text, you didn't remove the actual link. You'll need to remove the entire sentence "To learn more about when..."
![image](https://user-images.githubusercontent.com/37936169/233471344-1040b9a3-4231-4dc8-a8c9-cc6cea6e805c.png)

Reply:
fix